build.gradle

```diff
@@ -90,10 +90,12 @@ configure(projectsWithFlags('java')) {
         testRuntimeOnly libs.junit5.vintage.engine
     }
 
-    // Target Java 8.
-    tasks.withType(JavaCompile) {
-        if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
-            options.release = 8
+    // Target Java 8 except for spring-boot3
+    if (!project.name.startsWith("java-spring-boot3")) {
+        tasks.withType(JavaCompile) {
+            if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+                options.release = 8
+            }
         }
```

Comment:
Could we remove this block? According to what I understand, `--release` target is automatically set by `java.gradle`.

Reply:
I ran into issues with jgit6 when I removed this previously 😅   Let me try again

Reply:
I don't think this is a trivial issue 😅    Let me handle this in a separate issue and proceed with the release for now.

Reply:
Sounds good.